core/conversion/converters/converter_util.cpp

```diff
@@ -177,7 +177,11 @@ nvinfer1::ITensor* tensor_to_const(ConversionCtx* ctx, at::Tensor t) {
   tensor_id << reinterpret_cast<int*>(out);
 
   LOG_DEBUG(ctx->logger, "Freezing tensor " << tensor_id.str() << " as an IConstantLayer");
-  const_layer->setName(("[Freeze Tensor " + tensor_id.str() + " ]").c_str());
+  if (!name.empty()) {
+    const_layer->setName(name.c_str());
```

Comment:
Should this set the name of the tensor or the name of the layer? I think its easier if the message `[Freeze Tensor " + name + " ]` is there for debugging, but having an actual name for the tensor might be nice

Reply:
I think it sets the name of the layer. 
Yeah by default, it sets the one you mentioned `[Freeze Tensor " + name + " ]` 
This name argument is optional if anyone wants to set it to a more explicit name. 
For example, when you check the verbose log of TensorRT tactics, most of these constants are named `Freeze Tensor ....`
If anyone wants to find an explicit constant by name, it would be nice to have I thought.

Reply:
I mean the constant layer i think is always freezing a tensor, what i meant is we can have some conditional where its like 

```
if name:
    tensor_id = name
else: 
    tensor_id = int(&tensor)
const_layer.setName([Freeze tensor {}].(tensor_id))
```